cmd/training-operator.v1/main.go

```diff
@@ -165,8 +168,36 @@ func setupControllers(mgr ctrl.Manager, enabledSchemes controllerv1.EnabledSchem
 		cfg := mgr.GetConfig()
 		volcanoClientSet := volcanoclient.NewForConfigOrDie(cfg)
 		gangSchedulingSetupFunc = common.GenVolcanoSetupFunc(volcanoClientSet)
+		gvk := schema.GroupVersionKind{
```

Comment:
can we move the crd validation logic to common function whihc is used by both

Reply:
Done